qiskit/algorithms/evolvers/variational/var_qte.py

```diff
@@ -0,0 +1,235 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Variational Quantum Time Evolution Interface"""
+
+from abc import ABC
+from functools import partial
+from typing import Optional, Union, Dict, List, Callable
+
+import numpy as np
+from scipy.integrate import RK45, OdeSolver
+
+from qiskit import QuantumCircuit
+from qiskit.algorithms import EvolutionProblem
+from qiskit.circuit import Parameter
+from qiskit.providers import Backend
+from qiskit.utils import QuantumInstance
+from qiskit.opflow import (
+    StateFn,
+    CircuitSampler,
+    OperatorBase,
+    ExpectationBase,
+)
+from qiskit.utils.backend_utils import is_aer_provider
+from .solvers.ode.ode_function_factory import OdeFunctionFactory
+
+from .solvers.var_qte_linear_solver import (
+    VarQTELinearSolver,
+)
+from .variational_principles.variational_principle import (
+    VariationalPrinciple,
+)
+from .solvers.ode.var_qte_ode_solver import (
+    VarQTEOdeSolver,
+)
+
+
+class VarQTE(ABC):
+    """Variational Quantum Time Evolution.
+       https://doi.org/10.22331/q-2019-10-07-191
```

Comment:
Could you reference the paper as
```
[1]

**References:**

[1] Author, Title (year) `URL <link_to_paper>`_
```

That's what we usually use in the other algos/files 🙂

Reply:
I noticed that in many places in the codebase the following format is used:
```
References:

    [1] Author. (year). Title. `URL <link_to_paper>`_
```